src/styles/utiles/_variables.scss

```
@@ -0,0 +1,15 @@
+$dayBoxSize: 100px;
+$dayBoxColor: #eee;
+$dayBoxFontSize: 30px;
+$dayBoxPadding: 10px;
+$dayBoxGapSize: 1px;
+$calendarWidth: $dayBoxSize*7 + $dayBoxGapSize*6 + $dayBoxPadding*2;
```

As well u can put here spaces:)
```suggestion
$calendarWidth: $dayBoxSize * 7 + $dayBoxGapSize * 6 + $dayBoxPadding * 2;
```